samples/nrf_desktop/src/modules/dfu.c

```diff
@@ -0,0 +1,242 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <inttypes.h>
+
+#include <zephyr/types.h>
+#include <misc/byteorder.h>
+#include <flash_map.h>
+#include <dfu/mcuboot.h>
+#include <bluetooth/conn.h>
+
+#include "event_manager.h"
+#include "config_event.h"
+#include "ble_event.h"
+
+#define MODULE dfu
+#include "module_state_event.h"
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_CONFIG_CHANNEL_DFU_LOG_LEVEL);
+
+
+#define FLASH_PAGE_SIZE 0x1000
+
+#define DFU_TIMEOUT 2000
```

Comment:
add units, I prefer to use `K_MSEC(2000)`

Reply:
Fixed